packages/cli/src/ui/components/shared/text-buffer.test.ts

```diff
@@ -677,6 +677,54 @@ describe('useTextBuffer', () => {
       expect(getBufferState(result).cursor).toEqual([0, 2]);
     });
 
+    it('should handle Meta+leftArrow for moving word left', () => {
```

Comment:
Can you tweak the tests so they ate parameterized to test all of the key combinations now mapped to moving a word left and right?